### ISSUE TITLE
Explain somewhere how to make openWMail default mail client

### ISSUE THREAD

**Issue by luzpaz** [openWMail/openWMail#36]
* Which version of openWMail are you using? [e.g. 1.3.1]
Version: 1.0.0  
* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Manjaro Linux 17.0.5

I'm trying to figure out how to make openWMail the default email client that opens when clicking on email addresses. I found out that the command line trigger should be:
`/opt/openwmail/openWMail --mailto=%u` 
but this will only open openWMail and not compose an email with the address in it's To: field 
Any ideas?

**Comment by luzpaz** [comment]
bump

**Comment by MggMuggins** [comment]
Can you add command line args in electron? If we can figure out how to send a mailto address to GMail, then I think it's possible. I'm not familiar with either the electron, node, or GMail API's though, so I have no idea how that would be done...
How do you mean, "Command line trigger"? How does one go about configuring the default Mailto in Linux? I've never bothered with it...

**Comment by geekgonecrazy** [comment]
The desktop file should register it as an option for mail.   But might depend on your desktop environment to chose it as the application to launch for mail links

**Comment by luzpaz** [comment]
I'm on Manjaro 17.0.5 XFCE 4.12
I set both the XFCE and the browser to open mail via openWMail. 
They trigger but they don't put the email address directly in the To: field
For example: Open https://www.w3schools.com/TAGS/tryit.asp?filename=tryhtml_link_mailto and click the mailto link 
`<a href="mailto:someone@example.com?Subject=Hello%20again" target="_top">Send Mail</a>`
. I set Chrome to open mailto: links with openWMail. It opens openWMail but it does't put the email address in the To: field and doesn't put the Subject in the subject field.

**Comment by MggMuggins** [comment]
That's definetly an issue with WMail. The big question is whether or not there is a google API for doing that.

**Comment by luzpaz** [comment]
@MggMuggins https://developers.google.com/web/updates/2012/02/Getting-Gmail-to-handle-all-mailto-links-with-registerProtocolHandler

**Comment by geekgonecrazy** [comment]
yeah that works great for google chrome.  But we'd probably need to add something to see the mailto flag being called.  Take the fields parse them... and then pass them on to the active email account's webview.  So that probably makes a bit more complicated

**Comment by MggMuggins** [comment]
My guess is that it wouldn't be too hard. A handful of strings to pass around.

**Comment by luzpaz** [comment]
@MggMuggins this would be such a welcomed feature.

**Comment by celem** [comment]
More than "welcome", necessary.

**Comment by geekgonecrazy** [comment]
Please feel free to open a PR :). Extra development time is limited.
